components/demio/sources/new-participant/new-participant.mjs

```diff
@@ -0,0 +1,48 @@
+import demio from "../../demio.app.mjs";
+
+export default {
+  key: "demio-new-participant",
+  name: "New Participant",
+  description: "Emit new event for each participant in an event",
+  type: "source",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    demio,
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+    eventId: {
+      propDefinition: [
+        demio,
+        "eventId",
+      ],
+    },
+    dateId: {
+      propDefinition: [
+        demio,
+        "dateId",
+        (c) => ({
+          eventId: c.eventId,
+        }),
+      ],
+    },
+  },
+  async run({ $ }) {
+    const participants = await this.demio.getEventDateParticipants({
```

Comment:
I guess we are having getting all participants here as [doc](https://publicdemioapi.docs.apiary.io/#reference/reports/event-date-participants/event-date-participants) says, if `this.demio.getEventDateParticipants` retrieves all participants, can we add some logic to filter new participants?